Source/Core/Core/GeckoCode.h

```diff
@@ -40,6 +40,10 @@ bool operator!=(const GeckoCode& lhs, const GeckoCode& rhs);
 bool operator==(const GeckoCode::Code& lhs, const GeckoCode::Code& rhs);
 bool operator!=(const GeckoCode::Code& lhs, const GeckoCode::Code& rhs);
 
+// the currently active codes
```

Comment:
Commented out code shouldn't be left in.

Reply:
oh whoops, forgot to remove that.